packages/app/src/containers/LayerThemeInspector.tsx

```diff
@@ -33,6 +51,63 @@ export default memo(function LayerThemeInspector() {
     [sharedStyles],
   );
 
+  const onSelect = useCallback(
+    (value) => {
+      const style = sharedStyles.find((s) => s.do_objectID === value);
+
+      if (style !== undefined) {
+        dispatch('setLayerStyle', value, style.value);
+      } else {
+        dispatch('setLayerStyle', value, undefined);
+      }
+    },
+    [sharedStyles, dispatch],
+  );
+
+  const onAdd = useCallback(() => {
+    const styleName = prompt('New Style Layout Name');
+
+    if (styleName != null) {
+      dispatch('addLayerStyle', styleName, selectedLayers[0].style);
+    }
+  }, [selectedLayers, dispatch]);
+
+  const onRename = useCallback(() => {
+    if (selectedLayerStyleId !== undefined) {
+      const newName = prompt('Rename Layout style');
+
+      if (newName !== null) {
+        dispatch('setLayerStyleName', selectedLayerStyleId, newName);
+      }
+    } else {
+      alert('No Layer Selected');
+    }
+  }, [selectedLayerStyleId, dispatch]);
+
+  const onDetach = useCallback(() => dispatch('setLayerStyle'), [dispatch]);
+
+  const onUpdate = useCallback(
+    () =>
+      selectedLayerStyleId
```

Comment:
We can use `if (!selectedLayerStyleId) return` here. We typically don't want a ternary `a ? b() : undefined` , where one branch has a side effect and the other doesn't